integration-tests/test/basic-l1-l2-communication.spec.ts

```diff
@@ -58,7 +60,8 @@ describe('Basic L1<>L2 Communication', async () => {
         L1SimpleStorage.interface.encodeFunctionData('setValue', [value]),
         5000000
       )
-
+      await transaction.wait()
+      await env.relayXDomainMessages(transaction)
```

Comment:
We don't rely on the message relayer anymore, now we relay transactions manually.